README.md

```diff
@@ -14,58 +14,55 @@ SPDX-License-Identifier: CERN-OHL-S-2.0
     https://api.reuse.software/badge/github.com/Wakoma/nimble)](
     https://api.reuse.software/info/github.com/Wakoma/nimble)
 
-An open source, rapidly deployable, wireless mesh network.
 
-Anyone connected to the nimble can video and voice chat, stream videos, share files, build and run eLearning courses and website, create collaborative spreadsheets and documents, read e-books, play games, and so much more… completely offline.
 
-If a single internet connection is available, it can be plugged into the nimble to provide free or paid internet access to anyone on the network. The offline services and content are always available to users, regardless of internet availability.
-
-Anyone in the world can build their own nimble with locally sourced hardware and 3D-printed components.
+<p align="center"><img alt="nimble logo 256" src="https://raw.githubusercontent.com/Wakoma/wakoma-assets/b84a00ba43119ff617c64942332dc7b5860567f4/nimble-assets/logos/nimble%20logo%20256.svg" height="auto" width="256"></p>
```

Comment:
I'd move the logo into the repo rather than link to something in the issue as the link may go dead.

Reply:
Can do if you prefer. I made this repo to prevent dead links and save space in the main repo: https://github.com/Wakoma/wakoma-assets/tree/main/nimble-assets/logos

Reply:
I think we keep that repo as a good place to have lots of branding and other assets. I would still copy a version into this repo as we will need to serve it with the server so having a local copy is useful.